### PR TITLE
ventoy.Ventoy version 1.0.88

### LINKED ISSUE #95595: [Package Request]: Ventoy
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [ ] I have checked that the installer for this package is not a script file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Ventoy
Publisher Url: https://github.com/ventoy/Ventoy
Package Name: Ventoy
Description: Ventoy is an open-source tool to create bootable USB drives for ISO/WIM/IMG/VHD(x)/EFI files.
Package Version: 1.0.88
Installer URL: https://github.com/ventoy/Ventoy/releases

Current Package Identifier (for new versions):  N/A

### PR DESCRIPTION
**Pull request by afermar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

Resolves #95595

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95620)

### CODE REVIEW

**Comment by rodrigoborgesdeoliveira** [comment]
Closes #95595

File: manifests/v/ventoy/Ventoy/1.0.88/ventoy.Ventoy.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PublisherUrl: https://www.ventoy.net
+Author: ventoy
+PackageName: Ventoy
+PackageUrl: https://www.ventoy.net/en/index.html
+License: GPL 3
+ShortDescription: Open source tool to create bootable USB drive for ISO/WIM/IMG/VHD(x)/EFI files.
+Description: Ventoy is an open source tool to create bootable USB drive for ISO/WIM/IMG/VHD(x)/EFI files.
+Moniker: ventoy
+Tags:
+- live usb
```
Comment (Trenly) [inline manifests/v/ventoy/Ventoy/1.0.88/ventoy.Ventoy.locale.en-US.yaml:17]:
```suggestion
- live-usb
```

File: manifests/v/ventoy/Ventoy/1.0.88/ventoy.Ventoy.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - RelativeFilePath: ventoy-1.0.88\altexe\Ventoy2Disk_ARM64.exe
+  InstallerUrl: https://github.com/ventoy/Ventoy/releases/download/v1.0.88/ventoy-1.0.88-windows.zip
+  InstallerSha256: BE6EA0C14D89A47D8F6E158BCFD43EFDC1580DCFF9C2E0D249A1A1085B5A2548
+- Architecture: arm64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: ventoy-1.0.88\altexe\Ventoy2Disk_ARM64.exe
+  InstallerUrl: https://github.com/ventoy/Ventoy/releases/download/v1.0.88/ventoy-1.0.88-windows.zip
+  InstallerSha256: BE6EA0C14D89A47D8F6E158BCFD43EFDC1580DCFF9C2E0D249A1A1085B5A2548
```
Comment (Trenly) [inline manifests/v/ventoy/Ventoy/1.0.88/ventoy.Ventoy.installer.yaml:34]:
```suggestion
  InstallerSha256: BE6EA0C14D89A47D8F6E158BCFD43EFDC1580DCFF9C2E0D249A1A1085B5A2548
Commands:
- ventoy
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — COMMENTED** [review]
My apologies for missing this the first time; Whenever the packages are installed, they use the name of the file unless `PortableCommandAlias` is specified. This would mean that users would have to use `Ventoy2Disk_ARM64` to run ventoy if the command alias isn't specified

**Review by Trenly — APPROVED** [review]

**Pull request merged**